examples/src/main/java/alluxio/examples/UnderFileSystemCommonOperations.java

```diff
@@ -210,7 +214,13 @@ public void createOpenLargeTest() throws IOException {
     int offset = 0;
     int noReadCount = 0;
     while (offset < buf.length && noReadCount < 3) {
-      int bytesRead = inputStream.read(buf, offset, buf.length - offset);
+      int bytesRead;
+      try {
+        bytesRead = inputStream.read(buf, offset, buf.length - offset);
+      } catch (Exception e) {
+        LOG.info("Failed to read from file {}", testFile, e);
```

Comment:
Does logging make sense or should we just print to console?

Reply:
In console, if this will cause test to fail, we should print out. But here, read throws exception because of eventual consistency is kind of expected. 
All logging is write to user.log to make console look cleaner, otherwise object store will has tons of logging

Reply:
Makes sense, for this log message, let's use `e.toString`, otherwise this will print the entire stack trace which is generally reserved for `log.error`